integration_tests/dbt_project.yml

```
@@ -1,8 +1,9 @@
 name: 'pardot_integration_tests'
-version: '1.0.0'
+version: '1.1.0'
 profile: 'integration_tests'
 config-version: 2
```

We should add in the following configuration in here, so we can run validation testing on data sets in our development warehouse. 

```suggestion
config-version: 2

models:
  +schema: "pardot_{{ var('directed_schema','dev') }}"
```